src/shared/meson.build

```diff
@@ -465,8 +465,8 @@ libshared_deps = [threads,
                   libseccomp,
                   libselinux,
                   libzstd,
-                  libxz,
-                  libxenctrl]
+                  libxenctrl,
+                  libxz]
```

Comment:
hmm? this does not look right. libzstd should be last

Reply:
Indeed. Fixed.